elements/pl-order-blocks/pl-order-blocks.css

```diff
@@ -0,0 +1,116 @@
+
+.connectedSortable {
+  /* This style applies to the MCQ options table, on the left hand side*/
+  border: 2px solid #A9A9A9; /* High visibility borders for accessibility */
+  border-radius: 4px;
+  background-color: #efefff; /* Pale Blue */
+
+  width: 100%;
+
+  margin: 0 40px 0 10px;
+
+  list-style-type: none;
+  padding: 5px 0 20px 0;
+  float: left;
+}
+
+.horizontalLayout {
+  /* This style applies to the DIV that holds each dropzone column */
+  /* This particular style renders the dropzones in a single row horizontally */ 
+  width: 46%;
+  margin-right: 20px;
+}
+
+.verticalLayout {
+  /* This style applies to the DIV that holds each dropzone column */
+  /* This particular style renders the dropzones in a single column vertically */ 
+  width: 55%;
+  margin-right: 20px;
+  margin-bottom: 20px;
+}
+
+
+.dropzone {
+  /* This style applies to answer dropzone */
+  background-color: #FFA; /* Yellow */
+  min-height: 30px;
+}
+
+
+.connectedSortable li, .dropzone li{
+  /* This style applies to the answer draggable tiles themselves */
+  background-color: #EFEFEF; /* Grey */
+  border: 2px solid #A9A9A9;
+  border-radius: 4px;
+  margin: 0 10px 10px 10px;
+  padding: 5px;
+  font-size: 1.2em;
+
+  display: block;
+
+  overflow: hidden;
+  text-overflow: ellipsis; /* Show overflow text as ... without mouse over */
```

Comment:
If it's helpful, here is some CSS for wrapping text around instead of using ellipsis, but I think either is totally fine: https://github.com/ace-lab/berk-parsons/blob/master/app/static/vendors/parsons/parsons.css#L34-L47

![image](https://user-images.githubusercontent.com/8859633/98585545-4f7fd980-227c-11eb-97a7-9d37b2604c0b.png)